examples/data-objects/inventory-app/src/schema.ts

```diff
@@ -3,24 +3,52 @@
  * Licensed under the MIT License.
  */
 
-import { FieldKinds, SchemaAware, SchemaBuilder, ValueSchema } from "@fluid-experimental/tree2";
+import {
+	AllowedUpdateType,
+	FieldKinds,
+	SchemaAware,
+	SchemaBuilder,
+	ValueSchema,
+} from "@fluid-experimental/tree2";
 
 const builder = new SchemaBuilder("inventory app");
-export const float64 = builder.leaf("number", ValueSchema.Number);
-export const string = builder.leaf("string", ValueSchema.String);
+// REV: I'm not sure why this is necessary...
+const float64 = builder.leaf("number", ValueSchema.Number);
+const string = builder.leaf("string", ValueSchema.String);
```

Comment:
@CraigMacomber will know the exact plan for declaring primitives in schema now/short-term/long-term as well as some of the other questions you have below, but it's already true that you can import existing libraries and reference those rather than re-declare:

```
import { leaf } from "@fluid-experimental/tree2";

const builder = new SchemaBuilder("inventory app", leaf.primitives);
const part = builder.struct("Contoso:Part-1.0.0", {
    name: SchemaBuilder.field(FieldKinds.value, leaf.string),
    quantity: SchemaBuilder.field(FieldKinds.value, leaf.number)
});
```

Reply:
`leaf.primitives` doesn't seem to be a valid third argument, but `leaf.library` seems ok?  I assume there's no significant downside of picking up a little more than I need in that library.